pandas/tests/arrays/integer/test_dtypes.py

```diff
@@ -294,3 +294,18 @@ def test_astype_boolean():
     result = a.astype("boolean")
     expected = pd.array([True, False, True, True, None], dtype="boolean")
     tm.assert_extension_array_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "func",
+    [
+        lambda s: s.tolist()[0],
+        lambda s: s.to_dict()[0],
+        lambda s: list(s.iteritems())[0][1],
+        lambda s: list(iter(s))[0],
+    ],
+)
+def test_conversion_methods_return_type_is_native(func):
```

Comment:
paramterize per all Int and UInt types

also add tests for float / bool

may need o move this to the base class

Reply:
Added the tests. Atm they're in:
```
pandas/tests/arrays/boolean/test_astype.py
pandas/tests/arrays/floating/test_astype.py
pandas/tests/arrays/integer/test_dtypes.py
```
Do you want them moved?